### ISSUE TITLE
Forestry compatibility =  bronze, tin and gear mold

### ISSUE THREAD

**Issue by ghost** [Esteemed-Innovation/Flaxbeards-Steam-Power#464]
I don't know if this would be possible but would be cool to have bronze alloy from forestry, tin , and gear mold support.

Bronze = 3 copper + 1 tin in the Crucible
Bronze would be durability of stone and damage of iron and speed of brass. 

Tin =  just straight up tin ingots
Tin tools would be durability of wood and damage of stone and faster than brass. 

Gear mold would be a whole new thing that appears when Forestry is installed where you can cast gears of copper, tin, iron etc. Only copper, bronze and tin gears would be useful though.

**Comment by elifoster** [comment]
I wouldn't add new toolsets for the materials, I would simply oredict the ingots to allow them to be used the existing tools from other mods.

**Comment by ghost** [comment]
@elifoster If you add Tin liquid and Bronze alloy liquids and melt recipes then it should support  any mod with tin or bronze . 

My MT3 script  does this like so

```
mods.fsp.Crucible.addBasicLiquid("tin", <Forestry:ingotTin>, null, <MagicBees:beeNugget:2>, 165, 203, 242);
mods.fsp.Crucible.addAlloyLiquid("bronze", <Forestry:ingotBronze>, null, null, "copper", 3, "tin", 1, 4, 247, 202, 96);
mods.fsp.Crucible.addOreMeltRecipe("ingotTin", "tin", 9);
mods.fsp.Crucible.addOreMeltRecipe("nuggetTin", "tin", 1);
mods.fsp.Crucible.addOreMeltRecipe("ingotBronze", "bronze", 9);
```

**Comment by elifoster** [comment]
Yeah I know. I'm just saying I'm not going to add a whole new set of tin and bronze tools.

**Comment by ghost** [comment]
Oh I understand! Np. What about gear mold? Would that be something you would be interested in? It would be a step above ingot. Maybe 5 ingot liquid = 1 gear.

**Comment by elifoster** [comment]
It's definitely something I've considered for mod compatibility. I'll have to look into how expensive gears are and all that stuff.

**Comment by Drullkus** [comment]
@elifoster Gears are worth 4 materials each. Applied Thaumagistics has Gears costing 3, plus plenty of vis.

**Comment by elifoster** [comment]
4 materials meaning 4 nuggets?

**Comment by Drullkus** [comment]
No, ingots. 4 ingotGold = 1 gearGold, 4 ingotInvar = 1 gearInvar, etc

**Comment by elifoster** [comment]
Ah, so 36 units, gotcha :+1: thanks

**Comment by ghost** [comment]
I'd say about 4/5 ingots per gear is standard.  Forestry is 5 ingots. IE2 is 4 ingots. You guys use   4 ingots and 1 nugget for a turbine. Is because the crafting shape is like a + sign in the crafting table.  Turbine could be made from a generic cast gear plus another part . 

This would cover pretty much anything basic. 

The only one other mold I can think of that could be useful is  a cylinder/piping/rod mold which would make something long  and narrow (hollow or not).  Generally this would be 2 ingots = 1  cylinder. For this mod you could use brass cylinders to make pistons, gun barrel etc. And in other mods you can use them to cast rods, or anything resembling that shape.
